x-pack/plugins/fleet/public/applications/fleet/components/search_bar.tsx

```diff
@@ -79,12 +80,12 @@ export const SearchBar: React.FunctionComponent<Props> = ({
       disableLanguageSwitcher={true}
       indexPatterns={
         indexPatternFields
-          ? [
+          ? ([
               {
                 title: indexPattern,
                 fields: indexPatternFields,
               },
-            ]
+            ] as unknown as DataView[])
```

Comment:
Just tested locally and looks like it works without the `unknown` 
```suggestion
            ] as DataView[])
```